autoload/fzf_mru/actions.vim

```diff
@@ -21,6 +21,14 @@ function! fzf_mru#actions#options() abort
   return options
 endfunction
 
+function! s:p(...)
+  let preview_window = get(g:, 'fzf_preview_window', '')
+    if len(preview_window)
+      return call('fzf#vim#with_preview', add(copy(a:000), preview_window))
```

Comment:
Whats the purpose of that: `add(copy(a:000), preview_window)` ?

Reply:
the add(copy....) code is copying the arguments passed to s:preview() and appending them to the preview_window variable.

Reply:
It looks like I changed the code somewhat from what it is in the fzf.vim code.  Here is the original s:p():
```
function! s:p(bang, ...)
  let preview_window = get(g:, 'fzf_preview_window', a:bang && &columns >= 80 || &columns >= 120 ? 'right': '')
  if len(preview_window)
    return call('fzf#vim#with_preview', add(copy(a:000), preview_window))
  endif
  return {}
endfunction
```